imports/ui/components/CardComponent.jsx

```diff
@@ -0,0 +1,119 @@
+        marginLeft: 'auto',
+        transition: theme.transitions.create('transform', {
+            duration: theme.transitions.duration.shortest,
+        }),
+    },
+    expandOpen: {
+        transform: 'rotate(180deg)',
+    },
+    avatar: {
+        backgroundColor: red[500],
+    },
+}));
+
+
+function CardComponent() {
+    const classes = useStyles();
+    const [expanded, setExpanded] = React.useState(false);
+
+    function handleExpandClick() {
+        setExpanded(!expanded);
+    }
+
+    return (
+        <Card className={classes.card}>
+            <CardHeader
+                avatar={
+                    <Avatar aria-label="Recipe" className={classes.avatar}>
+                        A
+                    </Avatar>
+                }
+                action={
+                    <IconButton>
+                        <MoreVertIcon/>
+                    </IconButton>
+                }
+                title="Bananas"
+                subheader="June 1, 2019"
+            />
+            <CardMedia
+                className={classes.media}
+                image="/static/images/cards/contemplative-reptile.jpg"
+                title="Paella dish"
+                // style={styles.media}
+            />
+            <CardContent>
+                <Typography variant="body2" color="textSecondary" component="p">
+                    This impressive banana is a perfect party dish and a fun meal to cook together with your
+                    guests. Add 1 cup of frozen peas along with the mussels, if you like.
+                </Typography>
+            </CardContent>
+            <CardActions disableSpacing>
+                <IconButton aria-label="Add to favorites">
+                    <FavoriteIcon/>
+                </IconButton>
+                <IconButton aria-label="Thumb Up">
+                    <ThumbUpIcon/>
+                </IconButton>
+                <IconButton aria-label="Thumb Down">
+                    <ThumbDownIcon/>
+                </IconButton>
+                {/*<IconButton aria-label="Share">*/}
+                    {/*<ShareIcon/>*/}
+                {/*</IconButton>*/}
+                <IconButton
+                    className={clsx(classes.expand, {
+                        [classes.expandOpen]: expanded,
+                    })}
+                    onClick={handleExpandClick}
+                    aria-expanded={expanded}
+                    aria-label="Show more"
+                >
+                    <ExpandMoreIcon/>
+                </IconButton>
+            </CardActions>
+            <Collapse in={expanded} timeout="auto" unmountOnExit>
+                <CardContent>
+                    <Typography paragraph>Location:</Typography>
+                    <Typography paragraph>
+                        Nearby!
+                    </Typography>
+
+                </CardContent>
+            </Collapse>
+        </Card>
+    );
+}
+
+export default CardComponent;
```

Comment:
I think you didn't run Prettier yet!  Webstorm have prettier too https://prettier.io/docs/en/webstorm.html but I haven't tried it.